src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String COLOR = "White";
+    private static final int RADIUS = 10;
+    private static final int NUMBER_FIGURE = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String color = colorSupplier.getRandomColor();
+        double number = random.nextDouble();
```

Comment:
The side should be random for each figure and now the value is the same for all figures.